pre/remap_restart/remap_restarts.py

```diff
@@ -24,33 +24,33 @@
 program_description = textwrap.dedent(f'''
       USAGE:
 
-      There are three ways to use this script to remap restarts.
+      This script provides three options to remap restart files:
 
-      1. Use questionary to convert template remap_params.tpl to
-         remap_params.yaml and then remap:
+      1. Use questionary to create a yaml config file (remap_params.yaml):
+           ./remap_questions.py
+         and then remap:
            ./remap_restarts.py
 
-      2. Use an existing config file to remap:
+      2. Use an existing yaml config file to remap:
            ./remap_restarts.py config_file -c my_config.yaml
 
-      3. Use command line's options as inputs:
-          ./remap_restarts.py command_line -ymdh 2004041421  ....
-          To see more command options, please use 
-          ./remap_restarts.py command_line -h
+      3. Use command line options:
+           ./remap_restarts.py command_line -ymdh 2004041421  ....
+         To see more command line options, please use 
+           ./remap_restarts.py command_line -h
+         Note that the questionary also generates command line inputs.
 
       There are three help commands:
-        ./remap_restarts.py -h
-        ./remap_restarts.py config_file -h
-        ./remap_restarts.py command_line -h
-
-      NOTE: Each individual script can be executed independently
-        1. remap_questions.py generates raw_answer.yaml
-        2. If command_line option is used, the command_line option is converted raw_answers.yaml
-        3. remap_params.py uses raw_answer.yaml and remap_params.tpl as inputs and generates remap_params.yaml
-        4. remap_upper.py uses remap_params.yaml as input for remapping
-        5. remap_lake_landice_saltwater.py uses remap_params.yaml as input for remapping
-        6. remap_catchANDcn.py uses remap_params.yaml as input for remapping
-        7. remap_analysis.py uses remap_params.yaml as input for remapping
+           ./remap_restarts.py -h
+           ./remap_restarts.py config_file -h
+           ./remap_restarts.py command_line -h
+
+      NOTE: Individual scripts can be executed independently:
+        1. remap_questions.py generates raw_answers.yaml.
+        2. If the command_line option is used with remap_restarts.py, the command line options are converted raw_answers.yaml.
+        3. remap_params.py uses raw_answer.yaml and remap_params.tpl as inputs and generates remap_params.yaml.
+        4. Sets of restarts can be remapped separately with remap_upper.py, remap_lake_landice_saltwater.py, 
+           remap_catchANDcn.py, and remap_analysis.py, all of which use remap_params.yaml as input for remapping.
```

Comment:
I'm a bit confused about `raw_answers.yaml`, which is generated by `remap_questions.py` according to item 1.  I thought that `remap_questions.py` generates `remap_params.yaml`.  Can `remap_params.yaml` be hidden from the users in these instructions?

Reply:
raw_answers.yaml is created by remap_questions.py and was passed to remap_params.py to generate remap_params.yaml.  The raw_answers.yaml is hidden now. It is used when debugging.

Reply:
I'm still a bit confused about the workings of the various scripts in this context, and how the "usage" text above can be edited without mentioning "raw_answers.yaml", given that "raw_answers.yaml" is hidden and used for debugging only.  If I understand correctly, currently `remap_questions.py` generates "raw_answers.yaml", which then makes it necessary to mention "raw_answers.yaml" in the "usage" text.  
Could we change `remap_questions.py` to generate "remap_params.yaml" by moving `params = remap_params(raw_config)` from `remap_restarts.py` to `remap_questions.py` (or maybe executing this in both scripts)?  
In any case, the "usage" text still needs editing, and I'm hoping it can be simplified by removing any mention of "raw_answers.yaml".

Reply:
The raw_answer.ymal was saved in a file before.  But it is not saved any more and the users won't know it. So we don't need to mention it any more.